### PR TITLE
i/7457: Add missing indication that image is linked

### LINKED ISSUE #7457: Missing indication that image is linked
## 📝 Provide a description of the improvement

Right now there is no major visual difference between a linked image and a "normal" image. Both look the same, and the only small difference is that the linked image has `pointer` cursor on `:hover` and after clicking it `LinkImageUI` component button is enabled.

NOTE: Linked image has a caption set.
image

Follow-up on: #7452

--
If you'd like to see this improvement implemented, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Feature (link): Added missing icon indication in the top-right corner of the image that informs the image is linked. Closes #7457.

Internal (theme-lark): Added the linked image indication styles.

### CODE REVIEW

**Comment by panr** [comment]
I decided to add the indication through the downcast instead of CSS pseudo-element.

**Comment by panr** [comment]
image

File: packages/ckeditor5-link/tests/linkimageediting.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ import normalizeHtml from '@ckeditor/ckeditor5-utils/tests/_utils/normalizehtml'
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
 import ImageCaptionEditing from '@ckeditor/ckeditor5-image/src/imagecaption/imagecaptionediting';
 
+const linkIconInditatorElement = '<span class="ck ck-link-image_icon"><svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">' +
```
Comment (Reinmar) [inline packages/ckeditor5-link/tests/linkimageediting.js:14]: That can be an import from the SVG file (the \<svg> part) so you won't need to update it when the SVG changes.

Comment (panr) [reply packages/ckeditor5-link/tests/linkimageediting.js:14]: 🤦 oh, right...

Comment (panr) [reply packages/ckeditor5-link/tests/linkimageediting.js]: Fixed it, but with additional dev comment.

**Pull request merged**